src/InsetOrderOptimizer.cpp

```diff
@@ -21,6 +21,19 @@ static int findAdjacentEnclosingPoly(const ConstPolygonRef& enclosed_inset, cons
     return -1;
 }
 
+void InsetOrderOptimizer::moveInside(coord_t dist)
+{
+    Point p = gcode_layer.getLastPlannedPositionOrStartingPosition();
+    if (PolygonUtils::moveInside(part.insets[0], p, dist) != NO_INDEX)
+    {
+        if (part.insets[0].inside(p))
+        {
+            gcode_layer.addTravel_simple(p);
+            gcode_layer.forceNewPathStart();
```

Comment:
Do you know why this forces a new path? Is there a new travel move afterwards?

Reply:
I just copied from LayerPlan::moveInsideCombBoundary().